crates/router/src/connector/paypal/transformers.rs

```diff
@@ -857,6 +942,120 @@ impl<F, T>
     }
 }
 
+impl<F>
+    TryFrom<
+        types::ResponseRouterData<
+            F,
+            PaypalThreeDsSyncResponse,
+            types::PaymentsSyncData,
+            types::PaymentsResponseData,
+        >,
+    > for types::RouterData<F, types::PaymentsSyncData, types::PaymentsResponseData>
+{
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(
+        item: types::ResponseRouterData<
+            F,
+            PaypalThreeDsSyncResponse,
+            types::PaymentsSyncData,
+            types::PaymentsResponseData,
+        >,
+    ) -> Result<Self, Self::Error> {
+        let status = storage_enums::AttemptStatus::foreign_from((
```

Comment:
As connector gives us PaypalThreeDsSyncResponse only if user authentication is incomplete, please set the AttemptStatus to AuthenticationPending rather than using try_from


Please also give comments on why we are going to set AuthenticationPending directly